360bc07

commit 360bc07
Author: Alain Delree <alain.delree@gmail.com>
Date:   Tue Aug 25 10:13:01 2026 +0200

    Supprime le secret .env local et nettoie .gitignore (retire .env, comptes_test.md)

diff --git a/.gitignore b/.gitignore
index ecb19f4..4209c0a 100644
--- a/.gitignore
+++ b/.gitignore
@@ -22,7 +22,6 @@ staticfiles/
 local_settings.py
 
 # Variables d'environnement
-.env
 .env.local
 
 # Éditeurs
@@ -34,4 +33,3 @@ local_settings.py
 # OS
 .DS_Store
 Thumbs.db
-comptes_test.md
